app/src/components/main/analytics/events/common/testItemPages/modalEventsCreators.js

```diff
@@ -373,6 +374,9 @@ const getOnClickEditorIconEvent = (page) => (isTIGroup) => {
     label: `${page}#${defectGroup}`,
   };
 };
+export const getOnClickIssueTicket = (pluginName) => {
+  return getClickIssueTicketEvent(MODAL_MAKE_DECISION)(pluginName);
+};
```

Comment:
Please move `getClickIssueTicketEvent` inside `getMakeDecisionModalEvents` function to use it in make decision modal directly from `SOME_PAGE.MAKE_DECISION_MODAL_EVENTS`